### PR TITLE
LP 139 Refactoring sizes

### PR DESCRIPTION
**Pull request by juanislas77**
Refactoring sizes of UI Components
https://devaptivist.atlassian.net/browse/LP-139

Resizing UI components using DesignSizes data class values

NOT IMAGE FOR REFERENCE

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 1]
```diff
@@ -217,13 +247,16 @@ fun LpFileButton(
         verticalAlignment = Alignment.CenterVertically,
         modifier = modifier
             .border(
-                border = BorderStroke(width = 1.dp, MaterialTheme.colorScheme.onPrimary),
-                shape = RoundedCornerShape(4.dp)
+                border = BorderStroke(
+                    width = LeasePertTheme.sizes.stroke,
+                    MaterialTheme.colorScheme.onPrimary
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: Please add the name of the property for the color.

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 2]
```diff
@@ -126,33 +126,38 @@ fun LpPostCard(
...
-        elevation = CardDefaults.cardElevation(1.dp),
+        border = BorderStroke(LeasePertTheme.sizes.stroke, MaterialTheme.colorScheme.onPrimary),
+        elevation = CardDefaults.cardElevation(LeasePertTheme.sizes.stroke),
     ) {
         Column(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(15.dp, 15.dp, 15.dp, 5.dp)
+                .padding(
+                    LeasePertTheme.sizes.smallSize,
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: please add the name of each property like: `start =  LeasePertTheme.sizes.smallSize`

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 3]
```diff
@@ -262,21 +267,26 @@ fun LPGenericElevatedCard(
...
         ),
-        shape = MaterialTheme.shapes.medium.copy(all = CornerSize(12.dp)),
+        shape = MaterialTheme.shapes.medium.copy(all = CornerSize(LeasePertTheme.sizes.midSmallSize)),
         modifier = Modifier
-            .width(312.dp)
-            .height(196.dp)
-            .padding(16.dp)
+            .width(LeasePertTheme.sizes.extraSize336)
+            .height(LeasePertTheme.sizes.extraSize199)
+            .padding(LeasePertTheme.sizes.smallSize)
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Please add the name of the properties padding in this case `padding(all = LeasePertTheme.sizes.smallSize)`

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt (hunk 1 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: Working on the comments

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 2 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Working on the comments

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 3 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Working on the comments

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 2 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Comment resolved

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 3 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Comment resolved

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt (hunk 1 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: Comment resolved

**Review by melissaquezada94 — APPROVED** [review]
Well done!

**Review by carlosph816 — APPROVED** [review]
Well done!

**Pull request merged**